deployability/modules/workflow_engine/task.py

```diff
@@ -59,6 +67,9 @@ def execute(self) -> None:
             if result.returncode != 0:
                 raise subprocess.CalledProcessError(returncode=result.returncode, cmd=result.args, output=result.stdout)
         except subprocess.CalledProcessError as e:
+            error_msg = e.stderr
+            if "KeyboardInterrupt" in error_msg:
+                raise KeyboardInterrupt(f"Error executing process task with keyboard interrupt")
```

Comment:
We need to set some style criteria, in this case, a sentence ended with a dot or not

Reply:
Done in [ebce158](https://github.com/wazuh/wazuh-qa/pull/4960/commits/ebce158066ecff2dc0cb82cc7f902dc54b6d489d)